semantic_conventions/resource/browser.yaml

```
@@ -0,0 +1,41 @@
+groups:
+  - id: browser
+    prefix: browser
+    brief: >
+        The web browser in which the application represented by the resource is running.
+        The `browser.*` attributes MUST be used only for resources that represent applications
+        running in a web browser (regardless of whether running on a mobile or desktop device).
+    attributes:
+      - id: brands
+        type: string[]
+        brief: 'Array of brand name and version separated by a space'
+        note: >
+          This value is intended to be taken from the
+          [UA client hints API](https://wicg.github.io/ua-client-hints/#interface)
+          (navigator.userAgentData.brands).
+        examples: [" Not A;Brand 99", "Chromium 99", "Chrome 99"]
```

These example values are of type string, but the attribute is of type string[]. Should it be the following?
```suggestion
        examples: [[" Not A;Brand 99", "Chromium 99", "Chrome 99"]]
```

---

I thought so too at first, but I think the `type` determines how the example is formatted. All other existing attributes of type `string[]` have examples written the same way (e.g. [process.command_args](https://github.com/open-telemetry/opentelemetry-specification/blob/main/semantic_conventions/resource/process.yaml#L59))